test/support/utils.h

```diff
@@ -792,14 +795,16 @@ transform_reduce_serial(InputIterator first, InputIterator last, T init, BinaryO
     return init;
 }
 
-static const char*
-done()
+int
+done(bool is_passed = true)
 {
-#if _PSTL_TEST_SUCCESSFUL_KEYWORD
-    return "done";
-#else
-    return "passed";
-#endif
+    if(is_passed){
+        ::std::cout <<"Passed\n";
```

Comment:
you missed "done"

Reply:
Should we use specify one output on success? For rng_testsuite tests we use Passed and for other tests we use done.

Reply:
Applied